library/src/main/java/com/google/android/exoplayer/MediaFormat.java

```diff
@@ -15,22 +15,25 @@
  */
 package com.google.android.exoplayer;
 
+import android.annotation.SuppressLint;
```

Comment:
Please revert the change of import order (so android ones are still below the com.google ones).